openwisp_monitoring/monitoring/tests/test_db_creation.py

```diff
@@ -6,23 +6,25 @@
 from django.test import TestCase
 from requests.exceptions import ConnectionError
 
-from ..utils import get_db
+from ..settings import TIMESERIES_DB
+from ..utils import TimeseriesDB
 
 
 def mock_create_database(self, db):
     raise ConnectionError
 
 
 class TestDatabase(TestCase):
-    @patch('openwisp_monitoring.monitoring.settings.INFLUXDB_DATABASE', 'test_db')
     @patch('openwisp_monitoring.monitoring.apps.MonitoringConfig.warn_and_delay')
     @patch('influxdb.client.InfluxDBClient.create_database', mock_create_database)
     def test_check_retry(self, mock):
+        TIMESERIES_DB['NAME'] = 'test_db'
         try:
             apps.get_app_config('monitoring').create_database()
         except ConnectionError:
             pass
-        get_db().drop_database('test_db')
+        TimeseriesDB.get_db().drop_database('test_db')
+        TIMESERIES_DB['NAME'] = 'openwisp2'
```

Comment:
Why is it necessary to change the name of the db twice if previously it wasn't?

Can you keep and adapt `@patch('openwisp_monitoring.monitoring.settings.INFLUXDB_DATABASE', 'test_db')` instead?